spec/features/carrier_spec.rb

```diff
@@ -51,6 +52,7 @@
       expect(page).to have_content('test carrier')
       expect(page).to have_content('babywearing')
       expect(page).to have_content('test model')
+      expect(page).to have_content(loan.checkout_volunteer)
```

Comment:
I could add all the columns here.  Didn't know if that was really needed.  wdyt?

Reply:
Just a display thing. Not a big deal unless it drops coverage.